src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/solvers/BlazeSolver.kt

```diff
@@ -202,6 +202,17 @@ object BlazeSolver {
                     event.partialTicks,
                     matrixStack
                 )
+                if (Skytils.config.lineToNextBlaze) {
+                    val secondLowestBlaze = orderedBlazes.getOrNull(1)?.blaze ?: return
+                    RenderUtil.draw3DLine(
+                        Vec3(lowestBlaze.posX, lowestBlaze.posY + 1.5, lowestBlaze.posZ),
+                        Vec3(secondLowestBlaze.posX, secondLowestBlaze.posY + 1.5, secondLowestBlaze.posZ),
+                        5,
+                        Skytils.config.lineToNextBlazeColor,
+                        event.partialTicks,
+                        UMatrixStack.Compat.get()
```

Comment:
Use the already created matrixstack
```suggestion
                        matrixStack
```